fastlane/lib/fastlane/plugins/template/.gitignore

```diff
@@ -6,3 +6,5 @@ Gemfile.lock
 /_yardoc/
 /doc/
 /rdoc/
+fastlane/README.md
```

Comment:
Feels redundant to the test in the gem's `README` — but we can also commit it, I have no preference.